.evergreen/config.yml

```
@@ -2267,13 +2240,24 @@ tasks:
       - func: bootstrap-mongo-orchestration
       - func: run-fuzz-tests
 
-  - name: "test-aws-lambda"
-    tags:
-      - latest
-      - lambda
+  - name: "test-aws-lambda-deployed"
     commands:
-      - func: bootstrap-mongo-orchestration
-      - func: run-deployed-aws-lambda-tests
+      - command: ec2.assume_role
+        params:
+          role_arn: ${LAMBDA_AWS_ROLE_ARN}
+          duration_seconds: 3600
+      - command: shell.exec
+        params:
+          working_dir: src/go.mongodb.org/mongo-driver
+          binary: bash
```

"binary" is not listed as a parameter for the "shell.exec" command. For the documentation, it looks like we should use "shell": 

> Defaults to sh if not set. Note that this is usually bash but is dash on Debian, so it's good to explicitly pass this parameter

```suggestion
          shell: bash
```